packages/braid-design-system/lib/components/Autosuggest/Autosuggest.docs.tsx

```diff
@@ -597,53 +597,51 @@ const docs: ComponentDocs = {
         <>
           <Text>
             If no suggestions are available and you’d like to provide messaging
-            to the user, you can provide an object with a{' '}
-            <Strong>message</Strong> property to the{' '}
-            <Strong>suggestions</Strong> prop.
+            to the user, you can provide it via the{' '}
+            <Strong>noSuggestionsMessage</Strong> prop.
+          </Text>
+          <Text>
+            The message can either be a single piece of text, or a more
+            structured prompt by providing an object containing{' '}
+            <Strong>text</Strong> and <Strong>description</Strong>.
```

Comment:
Super-duper nitpicky, but this wording feels weird to me. I think it's because `by providing` is here without another verb that specifies "what you get by providing an object". I would've expected something like "xyz can be achieved by providing an object...".

```suggestion
            A simple message can be shown by providing a single piece of text.
            Alternatively, a more structured prompt can be shown by providing
            an object containing <Strong>text</Strong> and <Strong>description</Strong>.
```

Would need to update the changeset messaging too.